### ISSUE TITLE
Listing headers twice

### ISSUE THREAD

**Issue by neet-h-patel** [ajshort/vscode-include-autocomplete#7]
Hello! Thank you for the great plugin, however, the headers are listed twice and it is confusing sometimes. Would this be because I also have the Microsoft C/C++ plugin installed?

Thank you

**Comment by ajshort** [comment]
I don't think so - have you added any additional paths to your include path? Are the headers in the current working directory or another location?
